From 71596c13ffa243f84b71072f1249782057a7bd19 Mon Sep 17 00:00:00 2001
From: YumeMichi <do4suki@gmail.com>
Date: Thu, 8 Feb 2018 17:32:55 +0800
Subject: [PATCH 05/12] sepolicy: Label mpctl_socket as data_file_type

* ERROR: The following types on /data/ must be associated
  with the "data_file_type" attribute: mpctl_socket

Change-Id: I2b1aa10646e455899d4e25162510fe9298408c44
Signed-off-by: YumeMichi <do4suki@gmail.com>
---
 vendor/common/file.te | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vendor/common/file.te b/vendor/common/file.te
index 76ba2f9f..c71ac939 100644
--- a/vendor/common/file.te
+++ b/vendor/common/file.te
@@ -114,7 +114,7 @@ type sysfs_poweron_alarm, sysfs_type, fs_type;
 type sysfs_mpdecision, fs_type, sysfs_type;
 type sysfs_rqstats, fs_type, sysfs_type;
 type sysfs_cpu_online, fs_type, sysfs_type;
-type mpctl_socket, file_type, mlstrustedobject;
+type mpctl_socket, file_type, data_file_type, mlstrustedobject;
 type mpctl_data_file, file_type, data_file_type;
 
 #Define the files used by lm
-- 
2.17.1

